### PR TITLE
Add notify_deploy to app

### PR DESCRIPTION
**Pull request by kyrias**

### CODE REVIEW

**Comment by lundberg** [comment]
Looks good except env state may not've been set, so use fallback i.e.
``` py
env.get('state', 'unknown')
```
or change the output to not contain state at all if not present in env.

**Comment by kyrias** [comment]
Added the fallback as it's the simplest, and amended the commit.

**Comment by andreif** [comment]
I am not sure about target branch (`master`). Shouldn't it be `develop`? There is a bunch of merge conflicts when one tries merging develop.

File: blues/application/tasks.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 from .providers import get_providers
 
 from .. import git
+from .. import slack
```
Comment (andreif) [inline blues/application/tasks.py]: `from .. import git, slack´ ?

Comment (joar) [inline blues/application/tasks.py]: LGTM, PEP-8 allows both styles for `from _ import _`  -- https://www.python.org/dev/peps/pep-0008/#imports

File: blues/application/tasks.py [hunk 2]
```diff
@@ -190,3 +191,23 @@ def generate_nginx_conf(role='www'):
 
     with open(conf_path, 'w+') as f:
         f.write(conf)
+
+def notify_deploy(role=None):
```
Comment (andreif) [inline blues/application/tasks.py]: Top-level functions and class definitions should have 2 blank lines before them, while class methods just 1, see https://www.python.org/dev/peps/pep-0008/#blank-lines

**Comment by lundberg** [comment]
Maybe it should be decorated as a task aswell and added to app.py __all__

**Comment by andreif** [comment]
@lundberg @joar @kyrias I was thinking about task, but was not sure if it's useful to run separately. Mostly because I missed that we don't have prev commit sha anymore!? I think we need to know what sha was before the deploy, no? Also, I think it's good to have info about all servers in slack. It could be messy if we would run dozens of them, but until then I think it's fine. For more instances we would need another deploy notification system which could group log messages by project.

**Comment by lundberg** [comment]
Sure, thinks it's good for now, maybe @task later, fix pep8 comment and its ready for merge

**Comment by kyrias** [comment]
@lundberg Do you want me to close this PR and open a new one against `develop` like @andreif said?

**Comment by lundberg** [comment]
@kyrias yes please use develop branch

**Comment by kyrias** [comment]
(Opened #30 against develop.)

**Pull request closed without merging**